addons/website/static/src/snippets/s_floating_blocks/options.xml

```diff
@@ -10,14 +10,16 @@
 
 <t t-name="website.s_floating_blocks.new_card">
     <section data-name="Card" class="s_floating_blocks_block s_col_no_resize position-sticky d-flex overflow-hidden py-5 o_cc o_cc1">
-        <div class="s_floating_blocks_block_grid row o_grid_mode gx-0 gy-2 flex-grow-1 w-100 h-lg-100 overflow-hidden align-self-start align-self-lg-auto" data-row-count="6">
-            <div class="o_grid_item g-height-4 g-col-lg-8 col-lg-8 order-lg-0" style="z-index: 1; grid-area: 1 / 1 / 5 / 9;">
-                <p class="lead">A great subtitle</p>
-                <h2 class="display-4-fs">Card Title</h2>
-            </div>
+        <div class="container-fluid align-self-end">
+            <div class="s_floating_blocks_block_grid row o_grid_mode overflow-hidden" data-row-count="8">
+                <div class="o_grid_item g-height-4 col-lg-8 order-lg-0" style="z-index: 1; grid-area: 1 / 1 / 5 / 9;">
+                    <p class="lead">A great subtitle</p>
+                    <h2 class="display-4-fs">Card Title</h2>
+                </div>
 
-            <div class="o_grid_item g-height-2 g-col-lg-3 col-lg-3" style="z-index: 2; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px; grid-area: 5 / 1 / 7 / 4;">
-                <a href="#" title="" role="button" class="btn btn-lg btn-secondary">Button</a>
+                <div class="o_grid_item g-col-lg-4 col-lg-4" style="z-index: 2; --grid-item-padding-y: 0px; --grid-item-padding-x: 0px; grid-area: 7 / 1 / 9 / 5;">
```

Comment:
Good catch from Soukéina: `g-height-2` missing here (our test won't catch it since it is in a static file)